lib/origen_debuggers/j_link.rb

```diff
@@ -245,6 +245,11 @@ def read32(data, options = {})
       alias_method :read_longword, :read32
       alias_method :read_32, :read32
 
+      # Read 32-bit chunks of data using given byte address
```

Comment:
Don't really like adding a new method for something like this, better to make the existing read32 method accept a single value or an array of values for the data arg